### PR TITLE
[Docs] add Chinese version of 2_new_data_model

### PR DESCRIPTION
**Pull request by noobying**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

This PR adds Chinese version of 2_new_data_model

## Modification

This PR adds Chinese version of 2_new_data_model

### CODE REVIEW

File: docs_zh-CN/2_new_data_model.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+            obj_count += 1
+
+    coco_format_json = dict(
+        images=images,
+        annotations=annotations,
+        categories=[{'id':0, 'name': 'balloon'}])
+    mmcv.dump(coco_format_json, out_file)
+
+```
+使用如上的函数，用户可以成功将标注文件转化为JSON格式，之后可以使用`CocoDataset`对模型进行训练和评测。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:198]: 中文 English 中文  中英文之间需要加入空格

File: docs_zh-CN/2_new_data_model.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+通过本文档，你将会知道如何使用自定义的数据集对预定义的模型进行推理，测试以及训练。我们使用[balloon dataset](https://github.com/matterport/Mask_RCNN/tree/master/samples/balloon) 作为例子来描述整个过程。
+
+如下是基本的步骤：
+
+1. 准备自定义数据集
+2. 准备配置文件
+3. 在自定义数据集上进行训练，测试和推理。
+
+## 准备自定义数据集
+MMDetection一共支持三种形式应用新数据集：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:12]: 中文 English 中文
**中英文需要增加空格**

File: docs_zh-CN/2_new_data_model.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
+# 2: 在自定义数据集上进行训练
+
+通过本文档，你将会知道如何使用自定义的数据集对预定义的模型进行推理，测试以及训练。我们使用[balloon dataset](https://github.com/matterport/Mask_RCNN/tree/master/samples/balloon) 作为例子来描述整个过程。
+
+如下是基本的步骤：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:5]: 基本步骤如下： 
感觉更通顺

File: docs_zh-CN/2_new_data_model.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+3. 在自定义数据集上进行训练，测试和推理。
+
+## 准备自定义数据集
+MMDetection一共支持三种形式应用新数据集：
+1. 将数据集重新组织为COCO格式。
+2. 将数据集重新组织为一个中间格式。
+3. 实现一个新的数据集。
+
+我们通常建议使用前面两种方法，因为它们通常来说比第三种方法要简单。
+在本文档中，我们展示一个例子来说明如何将数据转化为COCO格式。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:18]: 如果想分段，需要多加一行空行

File: docs_zh-CN/2_new_data_model.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+**注意**：MMDetection现只支持对COCO格式的数据集进行mask AP的评测。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:19]: blank line

File: docs_zh-CN/2_new_data_model.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+    "iscrowd": 0 or 1,
+}
+
+categories = [{
+    "id": int,
+    "name": str,
+    "supercategory": str,
+}]
+```
+现在假设我们使用balloon dataset。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:55]: space

File: docs_zh-CN/2_new_data_model.md [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+下载了数据集之后，我们需要实现一个函数将标注格式转化为COCO格式。然后我们就可以使用已经实现的COCODataset类来加载数据后进行训练以及评测。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:56]: space

**Review by BIGWangYuDong — COMMENTED** [review]
There must be a space between Chinese and English, So, Please check your PR

File: docs_zh-CN/2_new_data_model.md [hunk 8]
```diff
@@ -0,0 +1,261 @@
...
+2. 将数据集重新组织为一个中间格式。
+3. 实现一个新的数据集。
+
+我们通常建议使用前面两种方法，因为它们通常来说比第三种方法要简单。
+
+在本文档中，我们展示一个例子来说明如何将数据转化为 COCO 格式。
+
+**注意**：MMDetection 现只支持对 COCO 格式的数据集进行 mask AP 的评测。
+
+所以对于要进行实例分割任务的用户而言，必须要将数据集转化为 COCO 格式。
```
Comment (ZwwWayne) [inline docs_zh-CN/2_new_data_model.md:24]: 所以用户如果要进行实例分割，只能将数据转成 COCO 格式

File: docs_zh-CN/2_new_data_model.md [hunk 9]
```diff
@@ -0,0 +1,261 @@
...
+
+categories = [{
+    "id": int,
+    "name": str,
+    "supercategory": str,
+}]
+```
+现在假设我们使用 balloon dataset。
+
+下载了数据集之后，我们需要实现一个函数将标注格式转化为 COCO 格式。然后我们就可以使用已经实现的 COCODataset 类来加载数据后进行训练以及评测。
```
Comment (ZwwWayne) [inline docs_zh-CN/2_new_data_model.md:61]: 后进行训练以及评测->并进行训练和评测

**Review by ZwwWayne — COMMENTED** [review]
Hi @noobying , thanks for the efforts. The overall PR looks great. Some tweaks need to be resolved to make it more smooth. I also suggest we read it again to make it more straightforward and easy to read.

File: docs_zh-CN/2_new_data_model.md [hunk 10]
```diff
@@ -0,0 +1,261 @@
...
+
+    coco_format_json = dict(
+        images=images,
+        annotations=annotations,
+        categories=[{'id':0, 'name': 'balloon'}])
+    mmcv.dump(coco_format_json, out_file)
+
+```
+
+使用如上的函数，用户可以成功将标注文件转化为JSON格式，之后可以使用 `CocoDataset` 对模型进行训练和评测。
```
Comment (AronLin) [inline docs_zh-CN/2_new_data_model.md:206]: Need space before and after `JSON`.

File: docs_zh-CN/2_new_data_model.md [hunk 11]
```diff
@@ -0,0 +1,261 @@
...
+
+## 训练一个新的模型
+
+为了使用新的配置方法来对模型进行训练，你只需要简单运行如下命令。
+
+```shell
+python tools/train.py configs/balloon/mask_rcnn_r50_caffe_fpn_mstrain-poly_1x_balloon.py
+```
+
+参考[情况 1](1_exist_data_model.md)来获取更多详细的使用方法。
```
Comment (AronLin) [inline docs_zh-CN/2_new_data_model.md:251]: Please pull the latest master branch of MMDetection and check whether this link can be linked to the Chinese version document.

File: docs_zh-CN/2_new_data_model.md (hunk 9 above)
Comment (noobying) [reply docs_zh-CN/2_new_data_model.md:61]: Thanks for the valueable review!

**Comment by AronLin** [comment]
There is something wrong about lint. Please refer to [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) and install the pre-commit to fix the code style.

**Comment by noobying** [comment]
```$ git commit -m 'try pre-commit'
[INFO] Installing environment for https://github.com/jumanjihouse/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('C:\\Ruby30-x64\\bin\\gem.CMD', 'install', '--no-document', '--no-format-executable', 'fake_gem__-0.0.0.gem')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    ERROR:  While executing gem ... (Gem::DependencyResolutionError)
        conflicting dependencies rubocop (>= 0.68.1) and rubocop (= 0.85.1)
      Activated rubocop-0.85.1
      which does not match conflicting dependency (>= 0.68.1)

      Conflicting dependency chains:
        fake_gem__ (= 0.0.0), 0.0.0 activated, depends on
        rubocop (= 0.85.1), 0.85.1 activated

      versus:
        fake_gem__ (= 0.0.0), 0.0.0 activated, depends on
        rubocop-rspec (= 1.40.0), 1.40.0 activated, depends on
        rubocop (>= 0.68.1)

      Gems matching rubocop (>= 0.68.1):
        rubocop-1.18.3

Check the log at C:\Users\Administrator.DESKTOP-EASJP9H\.cache\pre-commit\pre-commit.log
```
 I use the pre-commit on win10. And I install the config into hook. When i use the `git commit`, this error occurs. how can I  fix this problem? 
many thanks!

**Comment by ZwwWayne** [comment]
> ```
> [INFO] Installing environment for https://github.com/jumanjihouse/pre-commit-hooks.
> [INFO] Once installed this environment will be reused.
> [INFO] This may take a few minutes...
> An unexpected error has occurred: CalledProcessError: command: ('C:\\Ruby30-x64\\bin\\gem.CMD', 'install', '--no-document', '--no-format-executable', 'fake_gem__-0.0.0.gem')
> return code: 1
> expected return code: 0
> stdout: (none)
> stderr:
>     ERROR:  While executing gem ... (Gem::DependencyResolutionError)
>         conflicting dependencies rubocop (>= 0.68.1) and rubocop (= 0.85.1)
>       Activated rubocop-0.85.1
>       which does not match conflicting dependency (>= 0.68.1)
> 
>       Conflicting dependency chains:
>         fake_gem__ (= 0.0.0), 0.0.0 activated, depends on
>         rubocop (= 0.85.1), 0.85.1 activated
> 
>       versus:
>         fake_gem__ (= 0.0.0), 0.0.0 activated, depends on
>         rubocop-rspec (= 1.40.0), 1.40.0 activated, depends on
>         rubocop (>= 0.68.1)
> 
>       Gems matching rubocop (>= 0.68.1):
>         rubocop-1.18.3
> 
> Check the log at C:\Users\Administrator.DESKTOP-EASJP9H\.cache\pre-commit\pre-commit.log
> ```
> 
> I use the pre-commit on win10. And I install the config into hook. When i use the `git commit`, this error occurs. how can I fix this problem?
> many thanks!

This is because the installation of ruby does not succeed. You might need to refer to [this doc](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md#code-style) to install ruby, there is a section about it in the end. For windows environment, you might need fix more issue about it, you may try `conda install -c conda-forge ruby` first.
There are also two workarounds:
1. Use markdown lint tool in your IDE, for example, VSCode support markdown lint tool and you can use it to modify it in your IDE.
2. add '-n' in your command to skip the pre-commit hook and fix the issues in CI, which is not recommended as it waste the resources in CI.

File: docs_zh-CN/2_new_data_model.md [hunk 12]
```diff
@@ -1 +1,260 @@
...
+通过本文档，你将会知道如何使用自定义数据集对预先定义好的模型进行推理，测试以及训练。我们使用 [balloon dataset](https://github.com/matterport/Mask_RCNN/tree/master/samples/balloon) 作为例子来描述整个过程。
+
+基本步骤如下：
+
+1. 准备自定义数据集
+2. 准备配置文件
+3. 在自定义数据集上进行训练，测试和推理。
+
+## 准备自定义数据集
+MMDetection 一共支持三种形式应用新数据集：
```
Comment (ZwwWayne) [inline docs_zh-CN/2_new_data_model.md:11]: Usually there should be an empty line between title and contents

File: docs_zh-CN/2_new_data_model.md [hunk 13]
```diff
@@ -1 +1,260 @@
...
+我们通常建议使用前面两种方法，因为它们通常来说比第三种方法要简单。
+
+在本文档中，我们展示一个例子来说明如何将数据转化为 COCO 格式。
+
+**注意**：MMDetection 现只支持对 COCO 格式的数据集进行 mask AP 的评测。
+
+所以用户如果要进行实例分割，只能将数据转成 COCO 格式。
+
+### COCO标注格式
+用于实例分割的 COCO 数据集格式如下所示，其中的键（key）都是必要的，参考[这里](https://cocodataset.org/#format-data)来获取更多细节。
```
Comment (ZwwWayne) [inline docs_zh-CN/2_new_data_model.md:26]: empty line, see comments above.

**Comment by noobying** [comment]
> > ```
> > [INFO] Installing environment for https://github.com/jumanjihouse/pre-commit-hooks.
> > [INFO] Once installed this environment will be reused.
> > [INFO] This may take a few minutes...
> > An unexpected error has occurred: CalledProcessError: command: ('C:\\Ruby30-x64\\bin\\gem.CMD', 'install', '--no-document', '--no-format-executable', 'fake_gem__-0.0.0.gem')
> > return code: 1
> > expected return code: 0
> > stdout: (none)
> > stderr:
> >     ERROR:  While executing gem ... (Gem::DependencyResolutionError)
> >         conflicting dependencies rubocop (>= 0.68.1) and rubocop (= 0.85.1)
> >       Activated rubocop-0.85.1
> >       which does not match conflicting dependency (>= 0.68.1)
> > 
> >       Conflicting dependency chains:
> >         fake_gem__ (= 0.0.0), 0.0.0 activated, depends on
> >         rubocop (= 0.85.1), 0.85.1 activated
> > 
> >       versus:
> >         fake_gem__ (= 0.0.0), 0.0.0 activated, depends on
> >         rubocop-rspec (= 1.40.0), 1.40.0 activated, depends on
> >         rubocop (>= 0.68.1)
> > 
> >       Gems matching rubocop (>= 0.68.1):
> >         rubocop-1.18.3
> > 
> > Check the log at C:\Users\Administrator.DESKTOP-EASJP9H\.cache\pre-commit\pre-commit.log
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I use the pre-commit on win10. And I install the config into hook. When i use the `git commit`, this error occurs. how can I fix this problem?
> > many thanks!
> 
> This is because the installation of ruby does not succeed. You might need to refer to [this doc](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md#code-style) to install ruby, there is a section about it in the end. For windows environment, you might need fix more issue about it, you may try `conda install -c conda-forge ruby` first.
> There are also two workarounds:
> 
> 1. Use markdown lint tool in your IDE, for example, VSCode support markdown lint tool and you can use it to modify it in your IDE.
> 2. add '-n' in your command to skip the pre-commit hook and fix the issues in CI, which is not recommended as it waste the resources in CI.

Thanks for your advice. I fix it degrade the ruby3 to ruby2.

**Review by AronLin — APPROVED** [review]
LGTM.

**Review by BIGWangYuDong — APPROVED** [review]

File: docs_zh-CN/2_new_data_model.md [hunk 14]
```diff
@@ -1 +1,264 @@
...
+
+## 准备配置文件
+
+第二步需要准备一个配置文件来成功加载数据集。假设我们想要用 balloon dataset 来训练配备了 FPN 的 Mask R-CNN ，如下是我们的配置文件。假设配置文件命名为 `mask_rcnn_r50_caffe_fpn_mstrain-poly_1x_balloon.py`，相应保存路径为 `configs/ballon/`，配置文件内容如下所示。
+
+```python
+# 这个新的配置文件继承自一个基础（base）配置文件，只需要突出必要的修改部分即可
+_base_ = 'mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_1x_coco.py'
+
+# 我们需要对头（head）中的类别数量进行修改来匹配数据集的标注
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:219]: In order to maintain the unity of translation, we don't need to translate `head`

File: docs_zh-CN/2_new_data_model.md [hunk 15]
```diff
@@ -1 +1,264 @@
...
+```
+
+使用如上的函数，用户可以成功将标注文件转化为 JSON 格式，之后可以使用 `CocoDataset` 对模型进行训练和评测。
+
+## 准备配置文件
+
+第二步需要准备一个配置文件来成功加载数据集。假设我们想要用 balloon dataset 来训练配备了 FPN 的 Mask R-CNN ，如下是我们的配置文件。假设配置文件命名为 `mask_rcnn_r50_caffe_fpn_mstrain-poly_1x_balloon.py`，相应保存路径为 `configs/ballon/`，配置文件内容如下所示。
+
+```python
+# 这个新的配置文件继承自一个基础（base）配置文件，只需要突出必要的修改部分即可
```
Comment (BIGWangYuDong) [inline docs_zh-CN/2_new_data_model.md:216]: 原始配置文件

**Review by BIGWangYuDong — COMMENTED** [review]
Two more details need to modify

**Comment by noobying** [comment]
Thanks for your patience review. You guys really makes my day. Appreciate it!

**Pull request merged**